src/tests/baseservices/exceptions/unhandledHandler/ForeignThreadRevPInvokeUnhandled/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project (ForeignThreadRevPInvokeUnhandledNative)
```

Comment:
The directory name `unhandledHandler` has non-standard casing. Rename it to `UnhandledExceptionHandler`?